test/e2e/api_accounts_test.go

```diff
@@ -46,13 +46,21 @@ var _ = Context("Payments API Accounts", func() {
 
 	When("creating a new account", func() {
 		var (
-			e chan *nats.Msg
+			e           chan *nats.Msg
+			connectorID string
+			err         error
 		)
 
-		It("should be successful with v2", func() {
-			connectorID, err := installConnector(ctx, app.GetValue(), uuid.New(), 2)
+		BeforeEach(func() {
+			connectorID, err = installConnector(ctx, app.GetValue(), uuid.New(), 3)
 			Expect(err).To(BeNil())
+		})
+
+		AfterEach(func() {
+			uninstallConnector(ctx, app.GetValue(), connectorID)
+		})
```

Comment:
In the future it might not be a bad idea to implement a `t.CleanupTest()` type function that allows us to setup the AfterEach tear down functions in the constructors.